packages/website/vue.config.js

```diff
@@ -1,38 +1 @@
-module.exports = {
-  chainWebpack: (config) => {
-    config.resolve.extensions
```

Comment:
This configuration is what enables TSX support *and* our SVG loader.

Reply:
@vue/babel-preset-jsx

Reply:
The switch to the preset seems to work.